.github/workflows/unit_tests.sh

```diff
@@ -113,6 +118,9 @@ for phase in "${PHASES[@]}"; do
             ;;
         CLEANUP)
             info "Cleanup phase"
+            if [ -w /etc/machine-id.bak ]; then
```

Comment:
`if ! [ -f /etc/machine-id ] && [ -f /etc/machine-id.bak ];` ?

Reply:
better to check for write perm too